packages/vanilla-i18n/src/translationStore.ts

```diff
@@ -41,6 +57,11 @@ export function translate(str: string, defaultTranslation?: string): string {
         return translationStore[str];
     }
 
+    // report any untranslated strings when in translation debug mode
+    if (defaultTranslation === undefined && translationDebug()) {
```

Comment:
I'd even go a bit further. I think we should wrap the result even with the "default" provided.

```js
    let result = defaultTranslation !== undefined ? defaultTranslation : str;

    if (translationDebug()) {
    	result = "☢️☢️☢️" + result + "☢️☢️☢️";
    }
	return result;
```